components/suggest/src/schema.rs

```diff
@@ -193,6 +195,11 @@ CREATE TABLE IF NOT EXISTS dismissed_suggestions (
                 )?;
                 Ok(())
             }
+            19 => {
+                // Need to update new icons for Yelp especially in new schema.
+                let _ = clear_database(tx);
```

Comment:
Since only the Yelp table changed, can we do something like this instead?

1. If `yelp_custom_details` exists, get the current record from `yelp_custom_details` and then `DROP TABLE yelp_custom_details`
2. `CREATE TABLE yelp_custom_details` just like in the main schema SQL
3. If we got the record from the old table in step 1, insert it into the new table, using the old `icon_id` as the new `icon_light_theme_id`

Starting with schema version 16, I believe the idea is to avoid clearing the entire DB on upgrade if possible.

Sorry to add more work but It would also be nice if you could add a test for this migration step too.

Reply:
Okay, thanks!